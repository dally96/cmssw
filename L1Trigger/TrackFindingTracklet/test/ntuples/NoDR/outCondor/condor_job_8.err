************************************************************************************************************************
						 WARNING!
 The key phase2_realistic now points to 113X_mcRun4_realistic_v7.
 Usage of this key is discretionary and users are cautioned to use it at their own risk!
 This Global Tag contains tags suited for an Inner Tracker layout with rectangular (25x100um2) pixel cells (T15,T21).
 If this is not the geometry you are expecting to use, please consult:
 https://github.com/cms-sw/cmssw/blob/master/Configuration/AlCa/python/autoCondPhase2.py to retrieve the right key.
************************************************************************************************************************
16-Feb-2022 18:55:56 CET  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/92d5d7c4-11f2-4981-8162-11c25fe6527e.root
%MSG-w XrdAdaptorInternal:  file_open 16-Feb-2022 18:55:57 CET pre-events
Failed to open file at URL root://eoscms.cern.ch:1094//eos/cms/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/92d5d7c4-11f2-4981-8162-11c25fe6527e.root?xrdcl.requuid=25e33b81-ef50-4268-ab31-38a98ccfe8fe.
%MSG
%MSG-w XrdAdaptorInternal:  file_open 16-Feb-2022 18:55:57 CET pre-events
Failed to open file at URL root://eoscms.cern.ch:1094//eos/cms/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/92d5d7c4-11f2-4981-8162-11c25fe6527e.root?tried=&xrdcl.requuid=a2ad83ce-45ed-45c5-a9d3-86357a453620.
%MSG
16-Feb-2022 18:55:57 CET  Initiating request to open file root://xrootd-cms.infn.it//store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/92d5d7c4-11f2-4981-8162-11c25fe6527e.root
%MSG-w XrdAdaptor:  file_open 16-Feb-2022 18:55:58 CET pre-events
Data is served from lnl.infn.it instead of original site infn.it
%MSG
16-Feb-2022 18:56:03 CET  Successfully opened file root://xrootd-cms.infn.it//store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/92d5d7c4-11f2-4981-8162-11c25fe6527e.root
%MSG-w XrdAdaptor:  (NoModuleName) 16-Feb-2022 18:56:05 CET pre-events
Data is now served from lnl.infn.it, T2_UK_SGrid_RALPP instead of previous lnl.infn.it
%MSG
L1TrackNtupleMaker::beginJob
----- Begin Fatal Exception 16-Feb-2022 18:56:25 CET-----------------------
An exception of category 'FileWriteError' occurred while
   [0] Calling IOChannel::write()
Exception Message:
write() failed with system error 'Network dropped connection on reset' (error code 102)
----- End Fatal Exception -------------------------------------------------
L1TrackNtupleMaker::endJob
16-Feb-2022 18:56:25 CET  Closed file root://xrootd-cms.infn.it//store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/92d5d7c4-11f2-4981-8162-11c25fe6527e.root
TimeReport> Time report complete in 47.9087 seconds
 Time Summary: 
 - Min event:   1e+06
 - Max event:   0
 - Avg event:   0
 - Total loop:  0
 - Total init:  27.7695
 - Total job:   47.9087
 - EventSetup Lock:   0
 - EventSetup Get:   0
 Event Throughput: 0 ev/s
 CPU Summary: 
 - Total loop:  0
 - Total init:  8.82719
 - Total extra: 0
 - Total job:   9.7713
 Processing Summary: 
 - Number of Events:  0
 - Number of Global Begin Lumi Calls:  0
 - Number of Global Begin Run Calls: 0


=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 XrdAdaptor           -w (NoModuleName)                         1        1
    2 XrdAdaptor           -w file_open                              1        1
    3 XrdAdaptorInternal   -w file_open                              2        2
    4 TimeReport           -e AfterFile                              1        1
    5 Fatal Exception      -s                                        1        1
    6 fileAction           -s file_close                             1        1
    7 fileAction           -s file_open                              3        3

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 XrdAdaptor           pre-events                        
    2 XrdAdaptor           pre-events                        
    3 XrdAdaptorInternal   pre-events       pre-events       
    4 TimeReport           EndJob                            
    5 Fatal Exception      BeforeEvents                      
    6 fileAction           EndJob                            
    7 fileAction           pre-events       pre-events       pre-events

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 4                   4
Error                   1                   1
System                  5                   5

dropped waiting message count 0
Error in <TList::Clear>: A list is accessing an object (0x7ffdb155af30) already deleted (list name = TList)
terminate called after throwing an instance of 'edm::Exception'
  what():  An exception of category 'FileWriteError' occurred while
   [0] Calling IOChannel::write()
Exception Message:
write() failed with system error 'No such file or directory' (error code 2)



A fatal system signal has occurred: abort signal
The following is the call stack containing the origin of the signal.


Current Modules:

Module: none (crashed)

A fatal system signal has occurred: abort signal
Abort
