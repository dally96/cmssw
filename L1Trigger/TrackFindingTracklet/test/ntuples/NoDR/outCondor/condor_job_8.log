000 (801842.008.000) 02/16 18:54:26 Job submitted from host: <188.185.121.235:9618?addrs=188.185.121.235-9618+[2001-1458-d00-49--100-36e]-9618&alias=bigbird22.cern.ch&noUDP&sock=schedd_8027_6ecd>
...
040 (801842.008.000) 02/16 18:55:33 Started transferring input files
	Transferring to host: <128.142.58.79:9618?addrs=128.142.58.79-9618&alias=b7s03p1923.cern.ch&noUDP&sock=slot1_4_3594_0c4b_66224>
...
040 (801842.008.000) 02/16 18:55:33 Finished transferring input files
...
001 (801842.008.000) 02/16 18:55:34 Job executing on host: <128.142.58.79:9618?addrs=128.142.58.79-9618&alias=b7s03p1923.cern.ch&noUDP&sock=startd_2558_2759>
...
006 (801842.008.000) 02/16 18:55:43 Image size of job updated: 67244
	36  -  MemoryUsage of job (MB)
	36312  -  ResidentSetSize of job (KB)
...
006 (801842.008.000) 02/16 18:56:38 Image size of job updated: 2444884
	36  -  MemoryUsage of job (MB)
	36312  -  ResidentSetSize of job (KB)
...
040 (801842.008.000) 02/16 18:56:38 Started transferring output files
...
040 (801842.008.000) 02/16 18:56:38 Finished transferring output files
...
005 (801842.008.000) 02/16 18:56:38 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:18, Sys 0 00:00:03  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:18, Sys 0 00:00:03  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	25300  -  Run Bytes Sent By Job
	25473  -  Run Bytes Received By Job
	25300  -  Total Bytes Sent By Job
	25473  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :       63       17   1293272 
	   Memory (MB)          :       36     2000      2000 

	Job terminated of its own accord at 2022-02-16T17:56:38Z.
...
