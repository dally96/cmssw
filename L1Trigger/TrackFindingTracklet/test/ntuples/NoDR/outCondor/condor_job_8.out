/afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test /pool/condor/dir_31420 
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally/CN=546391038
issuer    : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
identity  : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
type      : RFC3820 compliant impersonation proxy
strength  : 2048
path      : /afs/cern.ch/user/d/dally/private/x509up_u141138
timeleft  : 09:18:52
key usage : Digital Signature, Key Encipherment
=== VO cms extension information ===
VO        : cms
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
issuer    : /DC=ch/DC=cern/OU=computers/CN=cms-auth.web.cern.ch
attribute : /cms/Role=NULL/Capability=NULL
timeleft  : 09:18:51
uri       : voms-cms-auth.app.cern.ch:443

subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally/CN=546391038
issuer    : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
identity  : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
type      : RFC3820 compliant impersonation proxy
strength  : 2048
path      : /afs/cern.ch/user/d/dally/private/x509up_u141138
timeleft  : 09:18:51
key usage : Digital Signature, Key Encipherment
=== VO cms extension information ===
VO        : cms
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
issuer    : /DC=ch/DC=cern/OU=computers/CN=cms-auth.web.cern.ch
attribute : /cms/Role=NULL/Capability=NULL
timeleft  : 09:18:50
uri       : voms-cms-auth.app.cern.ch:443

cmsRun /afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test/customise_L1TrackNtupleMaker.py inputFiles_clear inputFiles_load=/afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test/ntuples/NoDR/pFiles_8.txt outputFile=outputL1Ntuple_8.root print
Singletons:
  filePrepend        : 
                     - String to prepend location of all files
  maxEvents          : -1
                     - Number of events to process (-1 for all)
  outputFile         : outputL1Ntuple_8.root
                     - Name of output file (if needed)
  secondaryOutputFile: 
                     - Name of second output file (if needed)
  section            : 0
                     - This section (from 1..totalSections inclusive)
  tag                : 
                     - tag to add to output filename
  totalSections      : 0
                     - Total number of sections
Lists:
  inputFiles         : ['/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/92d5d7c4-11f2-4981-8162-11c25fe6527e.root' ]
                     - Files to process
  secondaryInputFiles: []
                     - Second group of files to process (if needed)

using geometry D76 (tilted)
Wed Feb 16 18:56:25 CET 2022
Thread 7 (Thread 0x2b60a0fb5700 (LWP 264)):
#0  0x00002b6047c3cb3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b6047c3cbcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b6047c3cc6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b604df00816 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#4  0x00002b604df008c9 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#5  0x00002b6047c36ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b6047f49b0d in clone () from /lib64/libc.so.6
Thread 6 (Thread 0x2b60a0db4700 (LWP 263)):
#0  0x00002b6047c3cb3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b6047c3cbcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b6047c3cc6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b604df00816 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#4  0x00002b604df008c9 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#5  0x00002b6047c36ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b6047f49b0d in clone () from /lib64/libc.so.6
Thread 5 (Thread 0x2b60a0bb3700 (LWP 262)):
#0  0x00002b6047c3cb3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b6047c3cbcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b6047c3cc6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b604df00816 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#4  0x00002b604df008c9 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#5  0x00002b6047c36ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b6047f49b0d in clone () from /lib64/libc.so.6
Thread 4 (Thread 0x2b60a09b2700 (LWP 261)):
#0  0x00002b6047c3de9d in nanosleep () from /lib64/libpthread.so.0
#1  0x00002b604df926ff in XrdSysTimer::Wait(int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdUtils.so.2
#2  0x00002b604dea2b0a in XrdCl::TaskManager::RunTasks() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#3  0x00002b604dea2c69 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdCl.so.2
#4  0x00002b6047c36ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b6047f49b0d in clone () from /lib64/libc.so.6
Thread 3 (Thread 0x2b60a07b1700 (LWP 260)):
#0  0x00002b6047f4a0e3 in epoll_wait () from /lib64/libc.so.6
#1  0x00002b604df8ced2 in XrdSys::IOEvents::PollE::Begin(XrdSysSemaphore*, int&, char const**) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdUtils.so.2
#2  0x00002b604df8978d in XrdSys::IOEvents::BootStrap::Start(void*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdUtils.so.2
#3  0x00002b604df91f68 in XrdSysThread_Xeq () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libXrdUtils.so.2
#4  0x00002b6047c36ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b6047f49b0d in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x2b606676a700 (LWP 253)):
#0  0x00002b6047c3e1d9 in waitpid () from /lib64/libpthread.so.0
#1  0x00002b604e29d387 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#2  0x00002b604e29e01a in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#3  0x00002b6047807af0 in std::execute_native_thread_routine (__p=0x2b6065941800) at ../../../../../libstdc++-v3/src/c++11/thread.cc:80
#4  0x00002b6047c36ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b6047f49b0d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x2b6049fc5180 (LWP 235)):
#0  0x00002b6047f3eddd in poll () from /lib64/libc.so.6
#1  0x00002b604e29d7b7 in full_read.constprop () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#2  0x00002b604e29e0ec in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#3  0x00002b604e2a159b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00002b6047e81387 in raise () from /lib64/libc.so.6
#6  0x00002b6047e82a78 in abort () from /lib64/libc.so.6
#7  0x00002b60477d2683 in __gnu_cxx::__verbose_terminate_handler () at ../../../../libstdc++-v3/libsupc++/vterminate.cc:95
#8  0x00002b60477de0a6 in __cxxabiv1::__terminate (handler=<optimized out>) at ../../../../libstdc++-v3/libsupc++/eh_terminate.cc:48
#9  0x00002b60477dd1a9 in __cxa_call_terminate (ue_header=ue_header@entry=0x2b60b039afe0) at ../../../../libstdc++-v3/libsupc++/eh_call.cc:54
#10 0x00002b60477ddad4 in __cxxabiv1::__gxx_personality_v0 (version=<optimized out>, actions=6, exception_class=5138137972254386944, ue_header=0x2b60b039afe0, context=<optimized out>) at ../../../../libstdc++-v3/libsupc++/eh_personality.cc:677
#11 0x00002b6047c247b3 in _Unwind_RaiseException_Phase2 (exc=0x2b60b039afe0, context=0x7ffdb155af60, frames_p=0x7ffdb155ae68) at ../../../libgcc/unwind.inc:64
#12 0x00002b6047c25016 in _Unwind_Resume (exc=0x2b60b039afe0) at ../../../libgcc/unwind.inc:241
#13 0x00002b6045f8cb1c in TFile::Close(char const*) [clone .cold] () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/external/slc7_amd64_gcc900/lib/libRIO.so
#14 0x00002b604d2a48ee in TStorageFactoryFile::~TStorageFactoryFile() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libIOPoolTFileAdaptor.so
#15 0x00002b604d2a4919 in TStorageFactoryFile::~TStorageFactoryFile() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libIOPoolTFileAdaptor.so
#16 0x000000000040f4d8 in std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_release() ()
#17 0x00002b60b3caab6c in edm::PoolOutputModule::~PoolOutputModule() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libIOPoolOutput.so
#18 0x00002b60b3caadaf in edm::PoolOutputModule::~PoolOutputModule() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libIOPoolOutput.so
#19 0x00002b609d105c03 in std::_Sp_counted_deleter<edm::one::OutputModuleBase*, std::default_delete<edm::one::OutputModuleBase>, std::allocator<void>, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreModules.so
#20 0x00002b609d106132 in std::_Sp_counted_ptr_inplace<edm::maker::ModuleHolderT<edm::one::OutputModuleBase>, std::allocator<edm::maker::ModuleHolderT<edm::one::OutputModuleBase> >, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/pluginFWCoreModules.so
#21 0x00002b6045692d7c in std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > >, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > > >::_M_erase(std::_Rb_tree_node<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#22 0x00002b6045692d06 in std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > >, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > > >::_M_erase(std::_Rb_tree_node<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#23 0x00002b6045692d06 in std::_Rb_tree<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > >, std::_Select1st<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > > >::_M_erase(std::_Rb_tree_node<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, edm::propagate_const<std::shared_ptr<edm::maker::ModuleHolder> > > >*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#24 0x00002b6045692dd6 in std::_Sp_counted_ptr<edm::ModuleRegistry*, (__gnu_cxx::_Lock_policy)2>::_M_dispose() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#25 0x00002b60455a9598 in std::default_delete<edm::Schedule>::operator()(edm::Schedule*) const [clone .isra.0] [clone .part.0] () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#26 0x00002b60455b0be2 in edm::EventProcessor::~EventProcessor() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_0_pre4/lib/slc7_amd64_gcc900/libFWCoreFramework.so
#27 0x000000000040b851 in (anonymous namespace)::EventProcessorWithSentry::~EventProcessorWithSentry() ()
#28 0x000000000040b649 in main ()
